page-content/Projects.html

```diff
@@ -4,5 +4,9 @@ <h3 class="page-title">Projects</h3>
     Here are all the projects that exist.
 </p>
 
+<div class="reload-button">
```

Comment:
I would maybe change the class name to something like "reload-button-div" or "reload-button-container" for clarity. Otherwise, with the class name "reload-button", I would assume that it is referring to the button itself, not the div.

Reply:
Resolved by commit e1776b6484b00f7dd9c7dc8867094b6699b9058f